src/gui/navigationpanehelper.cpp

```diff
@@ -102,7 +102,7 @@ void NavigationPaneHelper::updateCloudStorageRegistry()
                 auto title = folder->shortGuiRemotePathOrAppName();
                 // Write the account name in the sidebar only when using more than one account.
                 if (AccountManager::instance()->accounts().size() > 1) {
-                    title = title % " - " % folder->accountState()->account()->prettyName();
+                    title = title % " - " % folder->accountState()->account()->displayName();
```

Comment:
so the main issue with `Account::displayName()` is that the "user name" is not the one you set in Nextcloud but the user id you have used when logging in

we basically would want to use `Account::displayName()` but with the user name part being done with `Account::prettyName()`
https://github.com/nextcloud/desktop/blob/master/src/libsync/account.cpp#L208

I suggest that you write a new method in `Account` class to achieve that and add it to your PR

Reply:
Oh, i did not know that there can be difference between user name and user id.

I can try writing such a method. But this would be the first time programming in C++

Reply:
@SnejPro feel free to ask questions via the Talk desktop community room: https://cloud.nextcloud.com/call/d8k2r6p6
in general for home instances (my case for example) they do not differ
if you use ldap for example, most probably the user id will be an UUID while the display name will be "<First name> <Family name>"

Reply:
> @SnejPro feel free to ask questions via the Talk desktop community room: https://cloud.nextcloud.com/call/d8k2r6p6 in general for home instances (my case for example) they do not differ if you use ldap for example, most probably the user id will be an UUID while the display name will be " "

I've made a few changes. It should work now.